dash_labs/version.py

```diff
@@ -1 +1 @@
-__version__ = "0.4.0"
+__version__ = "1.0.0"
```

Comment:
```suggestion
__version__ = "1.0.0rc1"
```

That way you can test a pypi publish first :)

Reply:
good idea. done.